nixos/modules/programs/atop.nix

```
@@ -136,36 +136,37 @@ in
           packages = [ atop (lib.mkIf cfg.netatop.enable cfg.netatop.package) ];
           services =
             mkService cfg.atopService.enable "atop" [ atop ]
```

```suggestion
            (mkService cfg.atopService.enable "atop" [ atop ]) // {
```
we can just merge here

---

The return value from mkService is wrapped by lib.mkIf, so I don't think it can be updated without modification. Based on your suggestion, though, I pulled the lib.mkIf code out of mkService and replicated at each site where mkService is called and then restructured to use lib.mkMerge. Does that look better?